### PR TITLE
Update some documentation

### PR DESCRIPTION
**Pull request by RAOF**

### CODE REVIEW

File: include/wlcs/display_server.h [hunk 1]
```diff
@@ -35,49 +35,133 @@ typedef struct WlcsTouch WlcsTouch;
...
+    uint32_t version; /**< Version of the struct this instance provides */
+
+    /**
+     * Start the display server's mainloop.
+     *
+     * This should *not* block until the mainloop exits, which implies the mainloop
+     * will need to be run in a separate thread.
+     *
+     * This does not need to block until the display server is ready to process
+     * input, but it does need to be able to process other calls (notably
```
Comment (AlanGriffiths) [inline include/wlcs/display_server.h]: The referent of "it" is unclear. How about:
> ...but _display server_ does need to be able to process other calls...

**Comment by RAOF** [comment]
Ooops. Found the branch clarifying the docs!
bors r=AlanGriffiths

**Comment by RAOF** [comment]
bors r=AlanGriffiths

**Pull request merged**